src/utils/publication-utils.ts

```diff
@@ -0,0 +1,23 @@
+import { logRocketConsole } from 'services/shared';
+import { JOB_STATUS_SUCCESS } from 'services/supabase';
+import { hasLength } from './misc-utils';
+
+export const checkIfPublishIsDone = (payload: any): [boolean | null, any] => {
```

Comment:
This was broken out for stuff that was reverted - but leaving it stand alone makes sense.